templates/close_pr.txt

```diff
@@ -1,4 +1,4 @@
-subject: [GitHub] [%(repository)s] %(user)s closed pull request #%(pr_id)s: %(title)s
+subject: [%(repository)s] %(title)s (PR #%(pr_id)s)
```

Comment:
This format aligns with GitHub notification format.
I believe `[GitHub]` prefix adds no value, and it does clutter the subject, so I removed `[GitHub]`.

Reply:
I would also add, that for most projects I encounter the "repository" also doesn't add much value ... so moving this to the end, would show more of the title in email clients.

Reply:
And possibly adding a prefix of "Re: " to everything that doesn't start a thread might also help with the grouping?

Reply:
I think it would be better to avoid `Re: ` to avoid mess with locale-dependent `Re`.

>I would also add, that for most projects I encounter the "repository" also doesn't add much value

I see notifications from multiple GitHub projects:
* someone comments on an issue I create in one of the projects I use
* someone mentions me in another project
* someone comments on a PR I create

In all those cases having repository is helpful.
I'm fine with moving repository after the title though.

Reply:
Well I'm referring to the mailing-lists of projects ... https://lists.apache.org/list.html?dev@streampipes.apache.org
I don't think this tool has an effect on the notification github sends to you directly because you asked it to or because you commented on something.

Reply:
Exactly. If I subscribe to one or several mailing-lists of projects, then the very same thing would happen.
I might easily participate (as coder, reviewer, user) in several ASF projects, so having repository name in the mail subject is needed even in case a single "project mailing list" would likely have notifications from one repository only.

Once again: people do subscribe to several mailing lists, and ASF projects do have multiple repositories. So ditching repository-name from every notification would be devastating.